CouchbaseLite.xcodeproj/xcshareddata/xcschemes/CBL Swift Tests - iOS App.xcscheme

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "1140"
+   version = "1.3">
+   <BuildAction
+      parallelizeBuildables = "YES"
```

Comment:
is it okay to turn this on?

Reply:
I don't know. It seems to be OK.

Reply:
We have that check on other schemes as well so I will leave it.